chromadb/auth/__init__.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, system: System) -> None:
         super().__init__(system)
 
     @abstractmethod
-    def get_credentials(self) -> T:
+    def get_credentials(self, user_id: Optional[str] = None) -> T:
```

Comment:
is this coupling the abstract interface too tightly with the underlying provider we are introducing?

Reply:
Not necessarily. Right now we don't have use for it but I was thinking that in the future where creds can be fetched by some user_id can be helpful. Can remove for now and re-add when/if needed down the line.

Reply:
In hindsight, I think we can create an indirection here. Each implementing class will load it from settings, env var etc. This is not a runtime config that needs to change other than at startup. I doubt there will be a practical use case where multiple users will be used. Removing it.